Doxyfile

```diff
@@ -27,6 +28,10 @@ INLINE_SOURCES         = YES
 REFERENCED_BY_RELATION = YES
 REFERENCES_RELATION    = YES
 #---------------------------------------------------------------------------
+# Build related configuration options
+#---------------------------------------------------------------------------
+EXTRACT_STATIC         = YES
```

Comment:
Why would you make such change?

Reply:
To generate the document of static inline function of `cpu-utils.h`.